kbs/src/api/src/http/resource.rs

```diff
@@ -103,13 +103,17 @@ pub(crate) async fn get_resource(
             resource_description.resource_type,
             resource_description.resource_tag
         );
-        policy_engine
+        let (policy_valid, _extra_policy_output) = policy_engine
```

Comment:
Maybe `allow` could be a better name? As it is actually the evaluation result.

Surely, we need to emphasize in a document that the policy must have a value named `allow`. Or the evaluation will always return false.